### ISSUE TITLE
NPE in VertxEventBusMetrics on delivering message while handler registration is not finished yet

### ISSUE THREAD

**Issue by nolequen** [vert-x3/vertx-micrometer-metrics#81]
```
[ERROR] vert.x-worker-thread-28 - io.vertx.core.eventbus.impl.HandlerRegistration: Failed to handleMessage. address: some_address java.lang.NullPointerException
 at io.vertx.micrometer.impl.VertxEventBusMetrics.beginHandleMessage(VertxEventBusMetrics.java:90)
 at io.vertx.micrometer.impl.VertxEventBusMetrics.beginHandleMessage(VertxEventBusMetrics.java:33)
 at io.vertx.core.eventbus.impl.HandlerRegistration.deliver(HandlerRegistration.java:262)
 at io.vertx.core.eventbus.impl.HandlerRegistration.handle(HandlerRegistration.java:242)
 at io.vertx.core.eventbus.impl.EventBusImpl$InboundDeliveryContext.next(EventBusImpl.java:565)
 at io.vertx.core.eventbus.impl.EventBusImpl.lambda$deliverToHandler$5(EventBusImpl.java:524)
 at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
 at io.vertx.core.impl.WorkerContext.lambda$wrapTask$0(WorkerContext.java:34)
 at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
 at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
 at java.lang.Thread.run(Thread.java:748)
```
```
[ERROR] vert.x-worker-thread-5 - io.vertx.core.impl.ContextImpl: Unhandled exception java.lang.NullPointerException
 at io.vertx.micrometer.impl.VertxEventBusMetrics.endHandleMessage(VertxEventBusMetrics.java:98)
 at io.vertx.micrometer.impl.VertxEventBusMetrics.endHandleMessage(VertxEventBusMetrics.java:33)
 at io.vertx.core.eventbus.impl.HandlerRegistration.deliver(HandlerRegistration.java:271)
 at io.vertx.core.eventbus.impl.HandlerRegistration.handle(HandlerRegistration.java:242)
 at io.vertx.core.eventbus.impl.EventBusImpl$InboundDeliveryContext.next(EventBusImpl.java:565)
 at io.vertx.core.eventbus.impl.EventBusImpl.lambda$deliverToHandler$5(EventBusImpl.java:524)
 at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
 at io.vertx.core.impl.WorkerContext.lambda$wrapTask$0(WorkerContext.java:34)
 at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
 at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
 at java.lang.Thread.run(Thread.java:748)
```

**Comment by jotak** [comment]
This should already be fixed in 3.7.1, see #74 / https://github.com/eclipse-vertx/vert.x/issues/2892
Can you confirm?

**Comment by nolequen** [comment]
@jotak no.
Unfortunately I'm getting same errors on 3.7.1 too.
